### PR TITLE
Update weekly mailer content

### PR DESCRIPTION
**Pull request by swaathi**
## Notion card
No Notion card.

## Summary
I updated the grammar and content for both weekly update and invoice mailers. I also added a `monkey_patches` folder inside config that allows us to override Rails core libraries. 

## Preview
wu-text
wu-html
i-text
i-html

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Both manually tested and updated test cases. To see it in action use the mailer previews.

### Checklist:

- [x] I have manually tested all workflows
- [x] I have performed a self-review of my own code
- [x] I have added automated tests for my code

### CODE REVIEW

**Comment by supriya3105** [comment]
@swaathi the first email looks good. 

Regarding the 2nd one, users can pay by clicking on view invoice from the email itself. They cannot login into the platform to pay. Since all details are on the invoice email design we did not add text to it.

**Comment by swaathi** [comment]
> @swaathi the first email looks good.
> 
> Regarding the 2nd one, users can pay by clicking on view invoice from the email itself. They cannot login into the platform to pay. Since all details are on the invoice email design we did not add text to it.

Hi @supriya3105. So the text email that goes for invoice has that prompt to "logon", meaning to just visit the link and proceed. I didn't mean log "in". I read up on the use of the terminology [here](https://archives.cjr.org/language_corner/language_corner_101314.php). But let me know if the content is confusing, happy to reword it to "Please click on the link below to pay it now.".

**Comment by supriya3105** [comment]
> "Please click on the link below to pay it now.".

@swaathi Can we reword it as "Please click on the "View Invoice" button to pay now" .

**Comment by swaathi** [comment]
> @swaathi Can we reword it as "Please click on the "View Invoice" button to pay now" . 

@supriya3105 so it's not possible to add buttons to text emails. It must just be printed out. So the HTML email with the design is left intact, this is just the text version of the email.

**Comment by swaathi** [comment]
Hi @supriya3105 gentle reminder. :)

**Comment by swaathi** [comment]
@gowsik-ragunath appreciate if you can do a quick review on this!

**Review by akhilgkrishnan — APPROVED** [review]
@swaathi LGTM!. Thanks for working on this 👍

**Comment by swaathi** [comment]
Will merge this PR once Supriya and I align.

**Comment by supriya3105** [comment]
@akhilgkrishnan Can you remove the previous text email for invoice and raise updated PR .

**Comment by swaathi** [comment]
@supriya3105 I've removed both the unwanted text mail files. I'll merge this after CI passes.

**Comment by supriya3105** [comment]
> @supriya3105 I've removed both the unwanted text mail files. I'll merge this after CI passes.

ok.

**Comment by swaathi** [comment]
@akhilgkrishnan a JS spec has failed. Can you please check?

**Comment by akhilgkrishnan** [comment]
@apoorv1316 can you please check the cypress failure

**Comment by apoorv1316** [comment]
> @apoorv1316 can you please check the cypress failure

@swaathi @akhilgkrishnan Please rebase with develop.

**Comment by supriya3105** [comment]
@apoorv1316 can you test weekly reminder email and invoice email before we merge this PR.
